dist/install-common.sh

```diff
@@ -59,18 +59,22 @@ _adb_shell() {
     "$ADB" shell "$1"
 }
 
+_at_syscmd() {
```

Comment:
while we're at it, could we rewrite `setup_rootshell` to use this?

Reply:
Done.